### ISSUE TITLE
NullPointerException on research screen

### ISSUE THREAD

**Issue by aunoor** [joulupunikki/Phoenix#110]
Exception raise when try to open any research subcategory.
Patch in attachment.
[TechPanel.java.diff.zip](https://github.com/joulupunikki/Phoenix/files/713636/TechPanel.java.diff.zip)

**Comment by joulupunikki** [comment]
I can't reproduce this on Linux, Windows or FreeBSD. The first column of a tech row should always be something, either "Nothing" or a category name or a tech name, there should not be nameless rows. Are you using vanilla EFS1.4 or some mod ? The formatting of TECH.DAT is not obvious from reading just EFS1.4 files, mods tend to have odd DAT-files sometimes. If this is real, it could be deeper, something wrong with reading or processing of techs.

**Comment by aunoor** [comment]
I use vanilla 1.40 from dat_0527.exe
MD5 (./TECH.DAT) = a0f7ab178c787ad097450b09203134bc

**Comment by aunoor** [comment]
Yes, it can be reproduce only on Mac. Get 0.52.2 release jar and start it on Windows 8 and MacOs 10.10. Windows java version is 1.8.0_45 and Mac java version is 1.8.0_112.
Phoenix started under Windows goes normal, but under Mac exception was raised.
